includes/steem.php

```diff
@@ -351,13 +351,6 @@ public function _comment($parentAuthor, $parentPermlink, $author, $permlink, $ti
    */
   public function createPost($author, $title, $body, $tags = null, $app = null, $jsonMetadata = null, $permlink = null)
   {
-    if (empty($tags) && function_exists('get_option')) {
```

Comment:
let's keep the defensive preprocessing of tags to handle the corner case that tags is not provided, or provided as a string.